frontend/src/components/pages/ResidentDirectory/ResidentDirectoryTable.tsx

```diff
@@ -174,7 +190,11 @@ const ResidentDirectoryTable = ({
           <EditResident
             resident={editingResident}
             isOpen={isEditModalOpen}
-            toggleClose={handleEditClose}
+            toggleClose={() => {
+              handleEditClose()
+              getRecords(userPageNum)
```

Comment:
You'd want to pass this as a prop to the `EditResident` component and then call it right after the resident is successsfully updated